queries.sql

```diff
@@ -181,4 +181,116 @@ GROUP BY owners.full_name ORDER BY total DESC LIMIT 1;
 -------------+-------
  Melody Pond |     3
 (1 row)
-*/ 
\ No newline at end of file
+*/ 
+SELECT vets.name, animals.name, date_of_visit FROM vets JOIN visits ON vets.id=visits.vets_id
+JOIN animals ON animals.id= visits.animals_id WHERE vets.name ='William Tatcher' ORDER BY visits.date_of_visit DESC LIMIT 1;
+/*
+     name       |  name   | date_of_visit
+-----------------+---------+---------------
+ William Tatcher | Blossom | 2021-01-11
+(1 row)
+*/
+--How many different animals did Stephanie Mendez see?
```

Comment:
- [ ] I understand it feels nice to want to leave some comments to convey more meaning and description in our codebase. But it is often considered a good practice not to leave any especially when we are about to merge into the developed branch. It will be nice if you remove all commented code in this file 👍